lib/routes/oceanengine/index.js

```diff
@@ -0,0 +1,142 @@
+    template(__dirname + '/item-content.art', {
+        keyword,
+        queryListText,
+        queries: queryList.map((query) => ({
+            links: search_link_urls(encodeURIComponent(query)).map((url, index) => `<a href="${url}" rel="noopener noreferrer" target="_blank">${search_link_names[index]}</a>`),
+            key: query,
+        })),
+    });
+
+module.exports = async (ctx) => {
+    const now = dayjs();
+    const start_date = now.subtract(DEFAULT_FETCH_DURATION_MONTH, 'month').format('YYYYMMDD');
+    const end_date = now.format('YYYYMMDD');
+    const keyword = ctx.params.keyword;
+    if (!keyword) {
+        throw Error('Invalid keyword');
+    }
+    if (ctx.params.channel && !['douyin', 'toutiao'].includes(ctx.params.channel)) {
+        throw Error('Invalid channel。 Only support `douyin` or `toutiao`');
+    }
+
+    const channel = ctx.params.channel === 'toutiao' ? 'toutiao' : 'aweme'; // default channel is `douyin`
+    const channelName = ctx.params.channel === 'toutiao' ? '头条' : '抖音';
+
+    const link = `https://trendinsight.oceanengine.com/arithmetic-index/analysis?keyword=${keyword}&appName=${channel}`;
+
+    const item = await ctx.cache.tryGet(
+        link,
+        async () => {
+            const browser = await require('@/utils/puppeteer')();
+            const page = await browser.newPage();
+            await page.goto('https://trendinsight.oceanengine.com/arithmetic-index');
+            const res = await get_multi_keyword_hot_trend(page, keyword, start_date, end_date, channel);
+            browser.close();
+
+            const rawData = JSON.parse(res).data;
+            const data = decrypt(rawData).hot_list[0];
+
+            const search_top_point_list = data.search_top_point_list;
+            const search_hot_list = data.search_hot_list;
+            const search_average = parseInt(data.average.search_average);
+
+            return search_top_point_list.map(({ date, query_list }) => {
+                const search_index = parseInt(search_hot_list.find((listItem) => listItem.datetime === date).index);
+                const relative_percent = Math.round(((search_index - search_average) / search_average) * 100);
+                const icon = relative_percent > 0 ? '📈' : '📉';
+
+                const uniqueQueryList = query_list.filter((query) => query !== keyword);
+                const uniqueQueryListStr = uniqueQueryList.join(', ');
+
+                const content = create_content(keyword, uniqueQueryList, uniqueQueryListStr);
+
+                return {
+                    title: `${icon} ${keyword} ${relative_percent}% | ${uniqueQueryListStr}`,
+                    author: `巨量算数 - ${channelName}算数指数`,
+                    description: content,
+                    link,
+                    pubDate: parseDate(date),
+                    guid: `巨量算数 - ${channelName}算数指数 | ${keyword} - ${date}`,
+                };
+            });
+        },
+        CACHE_MAX_AGE
```

Comment:
Consider using configurable `contentExpire` from core config
https://github.com/DIYgod/RSSHub/blob/d581d778c693f253b81a71684841af69527b5cce/lib/config.js#L51
```js
const config = require('@/config').value;
// config.cache.contentExpire
```

You may need to pass an extra parameter `false` after cache max age `CACHE_MAX_AGE` for `cache.tryGet(key, getValueFn, maxAge, refresh)` to ensure the cache time will not be extended.

Reply:
The intent here is to extend the cache expire time to 6 hours due to the source website's low update frequency.

Reply:
> due to the source website's low update frequency

That's why `config.cache.contentExpire` existed

`config.cache.routeExpire` defined the cache ttl for frequently updated items.
`config.cache.contentExpire` defined the cache ttl for less-frequent/rarely updated items.

Reply:
Thanks. Changes are reflected [here](https://github.com/DIYgod/RSSHub/pull/9992/files#diff-9451ce3b67401576bf943d8de71b3ba08d262d8fdf0651fbb4b396aeded65073R9).